### DISCUSSION TITLE
Example yml file for GitHub to Digital Ocean using SSH

### DISCUSSION THREAD

**Discussion by jlhagger** [community/community#40847] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi, here is my yml file, unfortunately it's not working and I see a lot of people are wrestling with this. There's some great documentation on Digital Ocean and GitHub but none seem to cover this specficially or give an example .  It seems like a probably everyone new has to wrestle with?

name: Continuous Integration via SSH

on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Deploy site
        uses: actions/checkout@v3
        with:
         host: 137.184.57.52
         ssh-key: ${{secrets.SSH_KEY}}
         username: ${{secrets.SSH_USERNAME}}
         
         script: |
           cd ~/var/html_public/recipesgogogo.com
           git clone https://${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}:x-oauth-basic@github.com/jlhagger/recipesgogogo.git
           echo "Deployment successful to Digital Ocean"

**Comment by ossanna16** [comment]
Hi there @jlhagger and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by vitalykarasik** [reply]
Please share your error message

**Comment by jlhagger** [comment]
Dear Vitaly, I managed to fix it thanks – I found an appleboy script that helped me.

 

JL
